pilot/pkg/networking/plugin/authz/rbac.go

```diff
@@ -749,6 +782,43 @@ func convertToPrincipal(subject *rbacproto.Subject, forTCPFilter bool) *policypr
 	return &policyproto.Principal{Identifier: ids}
 }
 
+// convertBindingField converts an Istio first class field to Envoy RBAC config to nil if the field
```

Comment:
The comment is a little too general, we are converting to envoy RBAC config everywhere in this file. So make it more specific here.

Reply:
I changed the comment. Lmk if it's good now.